L1Trigger/Phase2L1Taus/BuildFile.xml

```diff
@@ -0,0 +1,39 @@
+<use   name="CommonTools/UtilAlgos"/>
+<use   name="CondFormats/DataRecord"/>
+<use   name="CondFormats/L1TObjects"/>
+<use   name="DataFormats/Common"/>
+<use   name="DataFormats/EcalDigi"/>
+<use   name="DataFormats/GeometryVector"/>
+<use   name="DataFormats/HcalDigi"/>
+<use   name="DataFormats/HepMCCandidate"/>
+<use   name="DataFormats/L1TCorrelator"/>
+<use   name="DataFormats/L1TParticleFlow"/>
+<use   name="DataFormats/L1TrackTrigger"/>
+<use   name="DataFormats/L1Trigger"/>
+<use   name="DataFormats/Math"/>
+<use   name="DataFormats/Phase2L1Taus"/>
+<use   name="DataFormats/Provenance"/>
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/MessageLogger"/>
+<use   name="FWCore/ParameterSet"/>
+<use   name="FWCore/PluginManager"/>
+<use   name="FWCore/ServiceRegistry"/>
+<use   name="FWCore/Utilities"/>
+<use   name="Geometry/CaloEventSetup"/>
+<use   name="Geometry/CaloGeometry"/>
+<use   name="Geometry/CaloTopology"/>
+<use   name="Geometry/CommonDetUnit"/>
+<use   name="Geometry/EcalAlgo"/>
+<use   name="Geometry/HcalTowerAlgo"/>
+<use   name="Geometry/Records"/>
+<use   name="PhysicsTools/TensorFlow"/>
+<use   name="tensorflow"/>
+<use   name="CommonTools/MVAUtils"/>
+<use   name="SimDataFormats/GeneratorProducts"/>
+<use   name="SimDataFormats/Track"/>
+<use   name="boost"/>
+<use   name="CLHEP"/>
+<use   name="root"/>
```

Comment:
This is nitpicking, so feel free to ignore it.

Could you remove the extra spaces between `use   name` (*i.e.* `use name`) and put the externals (`boost`, `root`, *etc.*) on top ?